documentation/modules/exploit/windows/http/manageengine_connectionid_write.md

```diff
@@ -0,0 +1,49 @@
+## Description
+
+This module exploits a vulnerability found in ManageEngine Desktop Central 9. When uploading a 7z file, the FileUploadServlet class does not check the user-controlled ConnectionId parameter in the FileUploadServlet class. This allows a remote attacker to inject a null bye at the end of the value to create a malicious file with an arbitrary file type, and then place it under a directory that allows server-side scripts to run, which results in remote code execution under the context of SYSTEM. This exploit was successfully tested on version 9, build 90109 and build 91084.
+
+**NOTE:** By default, some ManageEngine Desktop Central versions run on port 8020, but older ones run on port 8040. Also, using this exploit will leave debugging information produced by FileUploadServlet in file 'rdslog0.txt'.
+       
+## ManageEngine Desktop Central 9
+
+Desktop Central is integrated desktop and mobile device management software that helps in managing servers, laptops, desktops, smartphones, and tablets from a central location.It is used for automating your regular desktop management routines like installing patches, distributing software, managing your IT Assets, managing software licenses, monitoring software usage statistics, managing USB device usage, taking control of remote desktops, and more. It supports managing both Windows, Mac and Linux operating systems.
+
+## Prerequisites
+
+1. Start a Windows VM (such as Win 7)
+2. Install a vulnerable version of ManageEngine Desktop Central. This exploit was tested on Build 90109 and 91084. You can find the archive here: http://archives.manageengine.com/desktop-central/
```

Comment:
can you hotlink the 9109 and 91084 to the actual installer files in the archive? saves a few clicks.